MAUI/Calendar/getting-started.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
 title: Getting started with MAUI Calendar control | Syncfusion
-description: Learn here all about getting started with Syncfusion .NET MAUI Calendar (SfCalendar) control and its basic features.
+description: Learn here all about getting started with Syncfusion .NET MAUI Calendar(SfCalendar) control and its basic features.
 platform: maui
 control: SfCalendar
 documentation: ug
 ---
 
 # Getting started with .NET MAUI Calendar (SfCalendar)
-This section explains how to add the Calendar control. This section covers only the basic features needed to get started with Syncfusion Calendar.
+This section explains the steps required to add the Calendar control. This section covers only basic features needed to get started with Syncfusion Calendar.
 
 ## Creating an application using the .NET MAUI Calendar
 
 1. Create a new .NET MAUI application in Visual Studio.
 
-2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfCalendar` to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Calendar, and then install it.
+2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfCalendar to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Calendar and then install it.
```

Comment:
Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add `SfCalendar` to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Calendar, and then install it.

Reply:
Content changed